src/java/Test.java

```diff
@@ -27,6 +29,21 @@ public static void testStaticStack() throws Exception {
         System.out.println("Number of elements " + stack.count());
 
     }
+    public static void testSinglyLinkedList(){
+        SinglyLinkedList singlyLinkedList = new SinglyLinkedList();
+
+        singlyLinkedList.addElement(5);
+        singlyLinkedList.addElement(5);
+        singlyLinkedList.addElement(5);
+//        singlyLinkedList.addElement(5);
+//        singlyLinkedList.addElement(1);
+//        singlyLinkedList.addElement(6);
+        singlyLinkedList.displayList();
+        singlyLinkedList.removeDuplicatesUnsortedList();
+        System.out.println("---------------------------------------------");
+        singlyLinkedList.displayList();
+
+    }
```

Comment:
This is good, but we want to have multiple tests to cover the multiple scenarios. So, for example, when list is empty, this is one test case, when list has random elements, this is the second etc. Write everything that you can think of for each method (create separate helper methods testRemoveDuplicates and testRemoveDuplicatesUnsortedList). In order to do that, you can add removeAll method in the SinglyLinkedList class